pkg/splunk/enterprise/configuration_test.go

```diff
@@ -278,6 +280,90 @@ func TestSmartStoreConfigDoesNotFailOnIndexerClusterCRForCM(t *testing.T) {
 	}
 }
 
+func TestSmartstoreApplyIndexerClusterCreatesSmartStoreConfigMapOnValidConfig(t *testing.T) {
```

Comment:
Can't we make this a part of TestApplyIndexerCluster with the SmartStore spec added to the cr created + create namespace scoped secret key with smartstore keys updated? It would maintain the pattern currently existing?

Reply:
Smartstore config is optional,   so, having a separate test is a good choice here.

Reply:
Addressed.